todo_app/app.py

```diff
@@ -1,60 +1,33 @@
+from flask import request
 from flask import Flask, render_template,redirect
-from todo_app.data.trello_items import createcard, getcards
 from todo_app.flask_config import Config
-import requests
 import os
 from todo_app.ViewModel import ViewModel,Item
+import todo_app.data.mongo_items as mongo
+
 
 def create_app():
     app = Flask(__name__)
     app.config.from_object(Config())
 
     @app.route('/' , methods=['GET'])
-    def alltodoitems():
-        response = getcards()
-        cards=response.json() #changing the python objects to json
-        Allitems=[]
-        for card in cards:
-            if card['idList'] == os.getenv('TRELLO_LISTID1'):
-                status = 'To Do'
-            else:
-                status = 'Done'
-
-            item=Item(card['id'],card['name'],status)
-            Allitems.append(item)
-
-        todo_view_model = ViewModel(Allitems)
+    def alltodoitems():    
+        allitems = mongo.getitems()
+        todo_view_model = ViewModel(allitems)
         return render_template("index.html", view_model = todo_view_model)
 
 
     @app.route('/createnewcard' , methods=['POST'] )
     def createnewcard ():
-        response = createcard()
-        print(response.text)
+        name = request.form["title"]
+        mongo.createitem(name)
         return redirect('/')
 
     
 
     @app.route('/complete_item/<id>', methods=['POST'])
     def complete_item(id):
-        url =f"https://api.trello.com/1/cards/{id}"
-        params ={ 'key':os.getenv('TRELLO_KEY'),
-                'token':os.getenv('TRELLO_TOKEN'),
-                'idList':os.getenv('TRELLO_LISTID2')
-    }
-
-        headers = {
-            "Accept": "application/json"
-}
-
-        response = requests.request(
-            "PUT",
-            url,
-            headers=headers,
-            params=params
-    )
-        print(response.text)
+        mongo.changestatus(mongo._id,mongo.status)
```

Comment:
What does `mongo._id` or `mongo.status` refer to here?
I think the below would probably make more sense - since you've got the id set in a path variable (i.e. in the route & function definition) and this endpoint should always complete the card
```suggestion
        mongo.changestatus(id, "Done")
```

Reply:
Done